packages/backend-app-api/src/wiring/ServiceRegistry.ts

```diff
@@ -82,6 +82,13 @@ export class ServiceRegistry implements EnumerableServiceHolder {
     this.#implementations = new Map();
   }
 
+  add(serviceFactory: ServiceFactory) {
+    this.#providedFactories.set(
+      serviceFactory.service.id,
+      toInternalServiceFactory(serviceFactory),
+    );
+  }
```

Comment:
Tbh I think we should try avoid a mutating method here. Would be safer to have the service registry initialized with all factories instead